curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/660d8ca387f989d6b25a3343.md

```diff
@@ -16,13 +16,13 @@ To see the new result, click on the `"Add New Task"` button. Then add a title an
 Your `taskObj` should have an `id` property.
 
 ```js
-assert.match(code, /const\s+taskObj\s*=\s*\{\s*id/)
+assert.match(code, /const\s+taskObj\s*=\s*\{\s*("?|'?)id\1/)
 ```
 
 Your `id` property should have the value of the `titleInput.value`.
 
 ```js
-assert.match(code, /const\s+taskObj\s*=\s*\{\s*id:\s*titleInput\.value/)
+assert.match(code, /const\s+taskObj\s*=\s*\{\s*("?|'?)id\1\s*:\s*titleInput\.value/)
```

Comment:
```suggestion
assert.match(code, /const\s+taskObj\s*=\s*\{\s*("|'|)id\1\s*:\s*titleInput\.value\s*,?\s*\}/)
```
As @zxc-w mentioned it might be good to include closing curly bracket. I've thrown in optional comma as well.